### ISSUE TITLE
Inserting Images usting Strg+V shows Image 2 times in Editor

### ISSUE THREAD

**Issue by Wachert** [BookStackApp/BookStack#557]
### For Bug Reports

* BookStack Version *(Found in settings, Please don't put 'latest')*: 0.18.3
* PHP Version: PHP 5.6.30-0+deb8u1 (cli) (built: Feb  8 2017 08:50:21)
* MySQL Version: 5.5.55-0+deb8u1

##### Expected Behavior

Image should be shown only once.

##### Current Behavior

When you import an Image into the Editor(WYSIWYG) usting Strg+V (created using Snipping Tool from Microsoft in Windows 10), then the Image is shown 2 Times in the Editor instead of only one time.

**Comment by ssddanbrown** [comment]
Just for reference, Does not seem to appear on Ubuntu 17.04, Firefox or Chrome. Need to test further on Windows.

**Comment by Wachert** [comment]
I testet it a bit more, and in Chrome or Edge its not happening, but in Firefox (Version 57.0b*)
Tested in b5,b6,b7,b8

**Comment by ssddanbrown** [comment]
Since the last comment on this issue is relatively old, and since Firefox has had a few updates since this was opened, I'm going to close this. If the issue remains and is something you still require to be fixed please open a new issue, referencing this one.
